Fails to build and run tests on i386 and other 32bit platforms

### What is not working as expected?

Fails to build and run tests on i386 and other 32bit platforms

### What did you expect to happen?

the test should pass on all architectures

### How can we reproduce it?

```
GOOS=linux GOARCH=386 go test github.com/notaryproject/tspclient-go/internal/encoding/asn1/ber
# github.com/notaryproject/tspclient-go/internal/encoding/asn1/ber [github.com/notaryproject/tspclient-go/internal/encoding/asn1/ber.test]
internal/encoding/asn1/ber/ber_test.go:22:31: 0xFFFFFFFF + 8 (untyped int constant 4294967303) overflows int
FAIL	github.com/notaryproject/tspclient-go/internal/encoding/asn1/ber [build failed]
FAIL
```

### Describe your environment

go version go1.23.4 linux/amd64

### What is the version of your tspclient-go Library?

0.2.0

--- a/internal/encoding/asn1/ber/ber_test.go
+++ b/internal/encoding/asn1/ber/ber_test.go
@@ -19,7 +19,7 @@ import (
 )
 
 func TestConvertToDER(t *testing.T) {
-	var testBytes = make([]byte, 0xFFFFFFFF+8)
+	var testBytes = make([]byte, 0x7FFFFFFF)
 	// primitive identifier
 	testBytes[0] = 0x1f
 	testBytes[1] = 0xa0

